leapp/cli/upgrade/__init__.py

```diff
@@ -224,19 +225,15 @@ def upgrade(args):
     process_whitelist_experimental(repositories, workflow, configuration, logger)
     warn_if_unsupported(configuration)
     with beautify_actor_exception():
-        answerfile_path = cfg.get('report', 'answerfile')
         logger.info("Using answerfile at %s", answerfile_path)
         workflow.load_answerfile(answerfile_path)
         workflow.run(context=context, skip_phases_until=skip_phases_until, skip_dialogs=True)
 
     report_errors(workflow.errors)
     generate_report_files(context)
-
-    cfg = get_config()
-
     report_files = get_cfg_files('report', cfg)
     log_files = get_cfg_files('logs', cfg)
-    report_info(report_files, log_files, fail=workflow.failure)
+    report_info(report_files, log_files, answerfile_path, fail=workflow.failure)
```

Comment:
No answerfile is generated during an upgrade (see other comment).

Reply:
Good catch, should be fixed now.